.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Order is important; the last matching pattern takes the most precedence.
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence
+
+*   @ExpediaGroup/github-helpers-committers
```

Comment:
Might want to make following possible syntax are supported https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

Reply:
The [codowners-utils](https://www.npmjs.com/package/codeowners-utils) package should handle all supported syntax.